### ISSUE TITLE
Is this still viable?

### ISSUE THREAD

**Issue by sigzero** [flowchartsman/perl-ctypes#1]
Is this project "dead"? If so, was there another effort that does the same thing?

**Comment by flowchartsman** [comment]
Yes, this project never quite got off of the ground, and I have not used Perl in many years, so I'm out of touch enough that I'm afraid I can't provide a useful alternative, but I will archive the project now as a prelude to outright removal. Thanks for taking the time to file the issue, and sorry I couldn't be of further help.
